src/parse-result.js

```diff
@@ -63,6 +66,14 @@ export function namespace(options) {
     }
   }
 
+  [BooleanElement, StringElement, NumberElement, ObjectElement, ArrayElement].forEach((item) => {
```

Comment:
This means the `sourceMap` accessor won't be available on `Asset`, `DataStructure `, or `Extension` though.